safe/definitionsv4/hazard_classifications.py

```diff
@@ -431,6 +431,161 @@
     ]
 }
 
+
+cyclone_au_bom_hazard_classes = {
+    'key': 'cyclone_au_bom_hazard_classes',
+    'name': tr('Cyclone classes (AU - BOM)'),
+    'description': tr(
+        'The quinary <b>tropical cyclone</b> intensity classification '
```

Comment:
I think we should avoid the quinary etc. terminology - I think many users will not understand it and it is not common english. I know I started the trouble by using `ternary` once upon a time but it would be good to adopt more accessible language e.g. 'The five-class <b>tropical cyclone</b> ...'